.eslintrc

```diff
@@ -16,7 +17,19 @@
     "react/prefer-stateless-function": "off",
     "linebreak-style": [ 0 ],
     "import/extensions": "off",
-    "max-len": 0
+    "max-len": 0,
+    "promise/always-return": "error",
+    "promise/no-return-wrap": "error",
+    "promise/param-names": "error",
+    "promise/catch-or-return": "error",
+    "promise/no-native": "off",
+    "promise/no-nesting": "warn",
+    "promise/no-promise-in-callback": "warn",
+    "promise/no-callback-in-promise": "warn",
+    "promise/avoid-new": "warn",
+    "promise/no-new-statics": "error",
+    "promise/no-return-in-finally": "warn",
+    "promise/valid-params": "warn"
```

Comment:
@bongiovimatthew please confirm these rules are okay so that we have clean promises with return/catch functionalities. This way the next step would be to get rid of all the warnings/errors regarding promises so that they're handled the correct way

Reply:
This looks good!